docs/guides/algorithmiq-tem.ipynb

```diff
@@ -261,13 +262,15 @@
     "\n",
     "Name | Choices | Description  | Default\n",
     "-- | -- | -- | --\n",
-    "`max_bond_dimension` | int | The maximum bond dimension to be used for the tensor networks. | 500 |\n",
+    "`tem_max_bond_dimension` | int | The maximum bond dimension to be used for the tensor networks. | 500 |\n",
```

Comment:
Did the name change, or `max_bond_dimension` was always wrong?

Reply:
I think it was correct initially, then was changed, and not updated in the notebook.